doc/source/volshell.rst

```diff
@@ -180,15 +180,29 @@ used:
 
     layer = cc(mynewlayer.MyNewLayer, on_top_of = 'primary', other_parameter = 'important')
     with open('output.dmp', 'wb') as fp:
-        for i in range(0, 1073741824, 0x1000):
+        for i in range(0, 0x4000000, 0x1000):
             data = layer.read(i, 0x1000, pad = True)
             fp.write(data)
 
 As this demonstrates, all of the python is accessible, as are the volshell built in functions (such as `cc` which
 creates a constructable, like a layer or a symbol table).
 
+User Convenience
+----------------
+
+There are functions available that make often-done tasks easier, and generally provide a shell-like experience.  These can be listed using `help()` which, as already mentioned, is advertised when volshell starts.
+
 Loading files
--------------
+^^^^^^^^^^^^^
 
 Files can be loaded as physical layers using the `load_file` or `lf` command, which takes a filename or a URI.  This will be added
 to `context.layers` and can be accessed by the name returned by `lf`.
+
+Regex
+^^^^^
+
+It is easy to scan for some bytes or a pattern using `regex_scan` or `rx`.
+
+An optional size can be given for the displayed results as with the other fuctions (db, dw, dd, dq, etc).
+
+You can of course specify a different layer name as well.
```

Comment:
This should probably be `You can, of course, specify...` if we're happy with the colloquialism.

Reply:
Have changed it (better), maybe would be best without the "of course". Is commit squashing for pull requests implemented? Then can make small increments like this without undue credit.